protocol/accessibility/README.md

```diff
@@ -0,0 +1,93 @@
+
+* Prefer testing your local build using a screen reader such as [VoiceOver][voiceover] or [NVDA][nvda]
+* Prefer hiring assistive technology users to user test your product
+
+Quick checks
+------------
+
+### Design
+
+* Ensure all content's foreground color values meet the [minimum contrast ratio][color-contrast] for the background color they are placed over
+* Ensure all interactive content has distinct hover and focus states to help indicate interactivity
+* Ensure color is not the only way to determine meaning
+* Ensure interactive components use common UI affordances where applicable, to help users understand how they can be operated
+* Prefer icons and glyphs that don't rely on specialized knowledge to understand their meaning, unless being used in a domain-specific context
+* Prefer language that is [simple and direct][readability]
+* Ensure form inputs have labels that are visible in every state
+* Ensure link and button text is descriptive and distinct 
+* Prefer content that is broken into logical sections, with headings that explain the content that follows
+* Prefer text sizing that is set to 16px or larger
+* Ensure animation does not auto-play, can be paused, and avoids [vestibular and seizure triggers][vestibular-seizure]
+* Ensure video content has captions
+* Prefer larger interactive target sizes, with some space between grouped interactive controls
+
+### Development 
+
+* Ensure semantic markup is used to describe content
+* Ensure content does not disappear off the screen when zoomed
+* Ensure that interactive content can be tabbed to and activated using the keyboard, and that the tab order matches reading order
+* Ensure that heading elements are used, and that heading levels are placed in a logical order
+* Ensure that landmarks are used to describe the overall layout of the page or view
+* Ensure that alternative descriptions for image content are concise, descriptive, and use punctuation (`alt` attributes for images, `title` elements for SVGs)
+* Ensure labels are programmatically associated with their inputs
+* Prefer implementing a method to allow users to skip sections of repeated content
+* Each page or view has a unique title that describes the content it contains
+* The `title` attribute is only used to describe `iframe` element contents
+* Ensure that links are used to navigate to other locations and buttons are used to trigger actions
+* Ensure that focus is trapped inside of modal interactions
+* Ensure `fieldset` and `legend` elements are used to group related inputs and label them
+* Ensure form feedback messaging is programmatically associated with the relevant inputs
+
+Full audit
+----------
+
+When at all possible, use the guidelines in the basics and quick check sections to attempt to address accessibility in a proactive way. 
+
+If a thorough analysis needs to be performed, use the following workflow to perform a "deep" accessibility audit that checks against most WCAG criterion:
```

Comment:
Why the quotation marks around `"deep"`?

Criteria vs. criterion?

Reply:
Criterion is the official terminology the WCAG uses (which I think is kind of ironically a little too jargony, but helps with searching).

As for quotations around deep, I can remove them. Probably better to use a word like thorough or comprehensive.